WMF Framework/ExtensionViewController.swift

```diff
@@ -0,0 +1,30 @@
+import UIKit
+
+open class ExtensionViewController: UIViewController, Themeable {
+    public final var theme: Theme?
```

Comment:
Would it make sense to set the default theme here instead of having every widget VC fall back on its own?